examples/Carbon_Flux.ipynb

```diff
@@ -81,65 +223,411 @@
-    "* TIMESTAMP (Parsed out): YYYYMMDDHHMM\tISO timestamp – short format\n",
-    "* YEAR: Self explanatory\n",
-    "* DOY: Day of year.\n",
+    "* Good sanity checks:\n",
+    "    - lattitude encoded some structure, longitude does not"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualization\n",
     "\n",
-    "Here are some of the relevant FluxNet variables recorded at each site. The definitions may be found [here](http://fluxnet.fluxdata.org/data/fluxnet2015-dataset/fullset-data-product/):\n",
-    "   \n",
-    "* P_ERA : Precipitation, downscaled from ERA, linearly regressed using measured only site data\n",
-    "* TA_ERA: Air temperature, downscaled from ERA, linearly regressed using measured only site data\n",
-    "* PA_ERA: Atmospheric pressure, downscaled from ERA, linearly regressed using measured only site data\n",
-    "* SW_IN_ERA : Shortwave radiation, incoming, downscaled from ERA, linearly regressed using measured only site data (negative values set to zero)\n",
-    "* LW_IN_ERA : Longwave radiation, incoming, downscaled from ERA, linearly regressed using measured only site data\n",
-    "* WS_ERA: Wind speed, downscaled from ERA, linearly regressed using measured only site data\n",
-    "* SWC_F_MDS_1/2/3: Soil water content, gapfilled with MDS (numeric index “#” increases with the depth, 1 is shallowest)\n",
-    "* TS_F_MDS_1/2/3: Soil temperature, gapfilled with MDS (numeric index “#” increases with the depth, 1 is shallowest)\n",
-    "* VPD_ERA:  Vapor Pressure Deficit, downscaled from ERA, linearly regressed using measured only site data\n",
+    "Linear models work well *at one site* but this is confounded by\n",
     "\n",
-    "We'll use the above variables to predict:\n",
+    "* lat/lon\n",
+    "* day of year\n",
+    "* environment type\n",
     "\n",
-    "* NEE_CUT_USTAR50 : Net Ecosystem Exchange, using Constant Ustar Threshold (CUT) across years, from 50 percentile of USTAR threshold\n",
-    "* NEE_VUT_USTAR50: Net Ecosystem Exchange, using Variable Ustar Threshold (VUT) for each year, from 50 percentile of USTAR threshold\n",
+    "We want to generate some visualization that accounts for these 4 variables and helps generate some understanding.\n",
     "\n",
-    "Other variables should not be included as features, either because they are derived from NEE:\n",
+    "That is, these observations lie on some manifold. We want to learn the structure of that manifold, and visualize each observation on that manifold.\n",
     "\n",
-    "* GPP_DT_VUT_USTAR50: Gross Primary Production, from Daytime partitioning method, based on NEE_VUT_USTAR50\n",
-    "* GPP_DT_CUT_USTAR50: Gross Primary Production, from Daytime partitioning method, based on NEE_CUT_USTAR50\n",
+    "This will have the finding similar observations that have a similar structure between the indepedent variables (e.g., `P_ERA`) and depedent variables (the carbon flux measurement `y`).\n",
```

Comment:
Can't parse this sentence. Also: indepedent -> independent, depedent -> dependent

Reply:
@stsievert do you have any changes for this?